docs/.vitepress/vitepress/components/globals/parallax-home.vue

```diff
@@ -236,27 +238,28 @@ useEventListener(window, 'scroll', handleScroll)
 
   @media screen and (max-width: 959px) {
     .jumbotron {
-      display: none !important;
-    }
-
-    .mobile-banner {
-      margin-top: 10px;
-      display: inline-block;
+      .parallax-container {
+        width: 700px;
+        margin: 0 auto;
+      }
     }
   }
 
   @media (max-width: 768px) {
     .jumbotron {
       width: 50%;
-      display: flex;
-      margin: auto;
-      justify-content: center;
-      align-items: center;
+      display: none !important;
```

Comment:
Here this label is directly hidden, and the styles above or below can be removed.